src/modules/Remote.rb

```diff
@@ -104,6 +109,18 @@ def EnabledVncManager
       @mode == :vncmanager
     end
 
+    def IsWebVncEnabled
```

Comment:
well, I'm not such big expert in ruby but this ```web_vnc?``` is more according to the ruby style guide.

Reply:
Probably yes. I was just trying to replicate the style of the rest of the file. Function as `IsEnabled`, `IsDisabled`, `EnabledVncManager`.

Should I keep it as it is, or use `web_vnc?` for this new function, or maybe rename them all?

Reply:
Although the `web_vnc?` better fits the usual Ruby style I'd keep it as it is to have consistent names.